storybook/stories/avatar.stories.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { View } from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+import CenterView from './CenterView';
+import Avatar from '../../src/components/Avatar';
+import ArrowRight from '../../src/components/Icons/arrowRight';
+
+const photo = 'https://source.unsplash.com/mEZ3PoFGs_k/50x50';
+
+const iconStyles = {
+    color: 'white',
+    width: '100%',
+    height: '100%',
+};
+
+const viewStyle = {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+};
```

Comment:
As we did with the CenterView we should define a RowView and we can reuse it in other stories.
Then instead of use CenterView in the addDecorator method we use the RowView like:
```.addDecorator(getStory => <RowView>{getStory()}</RowView>);```

Reply:
Done